src/services/b2b.rs

```diff
@@ -9,192 +9,83 @@ use crate::errors::{MpesaError, MpesaResult};
 const B2B_URL: &str = "mpesa/b2b/v1/paymentrequest";
 
 #[derive(Debug, Serialize)]
-struct B2bPayload<'mpesa> {
-    #[serde(rename(serialize = "Initiator"))]
-    initiator: &'mpesa str,
-    #[serde(rename(serialize = "SecurityCredential"))]
-    security_credential: &'mpesa str,
-    #[serde(rename(serialize = "CommandID"))]
-    command_id: CommandId,
-    #[serde(rename(serialize = "Amount"))]
-    amount: f64,
-    #[serde(rename(serialize = "PartyA"))]
-    party_a: &'mpesa str,
-    #[serde(rename(serialize = "SenderIdentifierType"))]
-    sender_identifier_type: &'mpesa str,
-    #[serde(rename(serialize = "PartyB"))]
-    party_b: &'mpesa str,
+#[serde(rename_all = "PascalCase")]
+pub struct B2bRequest {
+    /// The credential/ username used to authenticate the transaction request
+    pub initiator: String,
+    pub security_credential: String,
+    pub command_id: CommandId,
+    pub amount: f64,
+    pub party_a: String,
+    pub sender_identifier_type: IdentifierTypes,
+    pub party_b: String,
     #[serde(rename(serialize = "RecieverIdentifierType"))]
```

Comment:
Do we still need this inner rename if he outer struct has the `rename_all="PascalCase"` attribute?

Reply:
Hehehe yeah, always has to be safaricom with the inconsistent API/docs.